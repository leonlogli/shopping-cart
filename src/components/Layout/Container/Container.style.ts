import styled from 'styled-components'
import { Container } from 'styled-bootstrap-grid'

export const StyledContainer = styled(Container)``

export default StyledContainer
