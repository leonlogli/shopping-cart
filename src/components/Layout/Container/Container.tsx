import React from 'react'
import { ContainerProps as BaseContainerProps } from 'styled-bootstrap-grid'

import StyledContainer from './Container.style'

export type ContainerProps = BaseContainerProps & {
  children: React.ReactNode
}

const Container = ({ children, ...other }: ContainerProps) => (
  <StyledContainer {...other}>{children}</StyledContainer>
)

export { Container }
export default Container
