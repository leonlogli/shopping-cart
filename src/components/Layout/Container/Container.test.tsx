import React from 'react'
import { mount } from 'enzyme'
import Container from './index'

describe('<Container />', () => {
  const wrapper = mount(
    <Container>
      <span>test</span>
    </Container>
  )

  it('should have the correct children', () => {
    expect(wrapper.find('span').length).toEqual(1)
  })

  it('should match a snapshot', () => {
    expect(wrapper).toMatchSnapshot()
  })
})
