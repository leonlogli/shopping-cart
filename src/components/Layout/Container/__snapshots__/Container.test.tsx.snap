// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Container /> should match a snapshot 1`] = `
<Container>
  <Styled(styled.div)>
    <div
      className="fWAStD"
      data-name="container"
    >
      <span>
        test
      </span>
    </div>
  </Styled(styled.div)>
</Container>
`;
